src/firebase/remoteConfigValuesDefaults.ts

```diff
@@ -73,4 +73,5 @@ export const REMOTE_CONFIG_VALUES_DEFAULTS: Omit<
   coinbasePayEnabled: false,
   showSwapMenuInDrawerMenu: false,
   shouldShowRecoveryPhraseInSettings: false,
+  createAccountCopyTestConfig: 'control',
```

Comment:
can we please use an enum for this? like how `superchargeButtonType` does it :)

Reply:
Yup, I made a type as `superchargeButtonType`